lib/setup/SetupClass.php

```diff
@@ -883,8 +918,8 @@ private function runWithPgEnv($sCmd)
      */
     private function pgExec($sSQL)
     {
-        if (!pg_query($this->oDB->connection, $sSQL)) {
-            fail(pg_last_error($this->oDB->connection));
+        if ($this->oDB->exec($sSQL) === false) {
+            fail($this->oDB->getLastError());
```

Comment:
Is that even necessary? I would expect that exec() throws and which leads to fail() anyways.